app/src/main/res/layout/activity_main.xml

```diff
@@ -6,13 +6,25 @@
     android:layout_height="match_parent"
     tools:context=".MainActivity">
 
+    <Button
+        android:id="@+id/count_button"
```

Comment:
why is this named "count_button" what's counting?

Reply:
Is not counting anything. I took it from an example of the constraint layout video. My bad. Fixed